.gitignore

```diff
@@ -9,6 +9,7 @@
 *~
 *.exe
 *-heap
+.hol/
```

Comment:
According to release notes, all temporary files now live in `.hol`, so I think we should remove `*Theory.*`, `.HOLMK`, etc., from `.gitignore`.

Reply:
Yes, I considered this at first, but it might be a problem for residual temporary files if you perform the upgrade in the same repo. IMO keeping them there is fairly benign, but I can remove them if you think it's cleaner.